config/default.json

```diff
@@ -1,14 +1,17 @@
 {
+  "PORT": 3000,
   "BULK": {
     "PORT": 3003
   },
-  "PORT": 3000,
```

Comment:
What do you think about using a separate config for the bulk?

Reply:
It was added by @vgenev with a PR against my fork. I'm not sure what would be the best approach. Would you suggest doing so and where should that config go (filename)?

Reply:
I can see the benefit of that. Also the Kafka Bulk config should be separated. And I don't like the current way we are having the both or each of the APIs up (npm script), but since the bulk api needs a lot more work, I would put that as a TODO.

Reply:
Guys, may we later decide it, after we have clarity on the entire implementation? We'll create another story for separating the config for the bulk. Moreover this a feature branch.